index.bs

```diff
@@ -1172,6 +1176,10 @@ Its value is «[=source type/navigation=] → 8, [=source type/event=] → 2».
 controls the maximum [=map/size=] of a [=trigger spec map=] for an
 [=attribution source=]. Its value is 32.
 
+<dfn>Max length per trigger context ID</dfn> is a positive integer that controls
+the maximum [=string/length=] of an [=attribution trigger=]'s [=attribution trigger/trigger context ID=],
+and an [=aggregatable report=]'s [=aggregatable report/trigger context ID=]. Its value is 64.
```

Comment:
Similar to https://github.com/WICG/attribution-reporting-api/pull/1113/files#r1400607818 I'd rather not explicitly say that this constant limits the length of the *report* field, but rather just the input field.

Reply:
Updated, thanks.